codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyHeaders.java

```diff
@@ -150,11 +150,18 @@ private HttpNames() { }
     SpdyHeaders setObject(CharSequence name, Object... values);
 
     @Override
-    SpdyHeaders set(TextHeaders headers);
+    SpdyHeaders set(Headers<? extends CharSequence> headers);
 
     @Override
-    SpdyHeaders setAll(TextHeaders headers);
+    SpdyHeaders setAll(Headers<? extends CharSequence> headers);
 
     @Override
     SpdyHeaders clear();
+
+    /**
+     * {@link #get(CharSequence)} and convert the result to a {@link String}.
+     * @param name the name of the header to retrieve
+     * @return the first header value if the header is found. {@code null} if there's no such header.
+     */
+    String getAsString(CharSequence name);
```

Comment:
@trustin - Should be done now. Check it out....let me know if you want other methods added.